File: legacy_scripts/check_webhooks.py
import json

with open('/tmp/current_wf.json') as f:
    wf = json.load(f)

if isinstance(wf, list):
    wf = wf[0]

print(f"Workflow ID: {wf.get('id')}")
for n in wf.get('nodes', []):
    if n['type'].startswith('n8n-nodes-base.webhook'):
        print(f"Webhook Node: {n['name']}")
        print(f"  Webhook ID: {n.get('webhookId')}")
        print(f"  Path: {n.get('parameters', {}).get('path')}")

File: legacy_scripts/check_webhooks.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('/root/.n8n/database.sqlite');

db.all("SELECT id, name, active FROM workflow_entity WHERE active=1;", (err, rows) => {
    if (err) throw err;
    console.log("ACTIVE WORKFLOWS:");
    console.table(rows);
    
    db.get("SELECT nodes FROM workflow_entity WHERE id = 'NO7NLZ5Kccp6jrOS';", (err, row) => {
        if (err) throw err;
        if(row) {
            const nodes = JSON.parse(row.nodes);
            const webhooks = nodes.filter(n => n.type === 'n8n-nodes-base.webhook');
            console.log("OLD WORKFLOW WEBHOOKS:");
            webhooks.forEach(w => console.log(w.name, w.parameters.path));
        }
    });

    db.get("SELECT nodes FROM workflow_entity WHERE id = 'A0QpoDzX559wzRXQ';", (err, row) => {
        if (err) throw err;
        if(row) {
            const nodes = JSON.parse(row.nodes);
            const webhooks = nodes.filter(n => n.type === 'n8n-nodes-base.webhook');
            console.log("NEW WORKFLOW WEBHOOKS:");
            webhooks.forEach(w => console.log(w.name, w.parameters.path));
        }
    });
});
